qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
+import re
+
+
+class KeysightB220X(VisaInstrument):
+    """
+    QCodes driver for B2200 / B2201 switch matrix
+    """
+
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\r', **kwargs)
+
+        self.add_function(name='clear_status',
+                          call_cmd='*CLS',
+                          docstring='Clears status register and error queue of the instrument.')
+
+        self.add_function(name='reset',
+                          call_cmd='*RST',
+                          docstring='Performs an instrument reset. Does not reset error queue!')
+
+        self.add_parameter(name='get_status',
+                           get_cmd='*ESR?',
+                           get_parser=lambda status_str: int(status_str),
```

Comment:
I think `get_parser=int` should work here.

Reply:
Oh yes of course 😅. I’ll change that.

Reply:
Done ✅